scripts/ast-patches/main.ts

```diff
@@ -1,12 +1,14 @@
-const path = require("path");
-const ts = require("typescript");
-const _ = require("lodash");
-const fs = require("fs-extra");
+import path from "path";
+import ts from "typescript";
+import _ from "lodash";
+import fs from "fs-extra";
 
-const astPatches = require("./ast-patches");
+import astPatches from ".";
```

Comment:
I don't know why, but this looks weird.. Is it importing from an `index.ts`? If yes, consider renaming the file as the entry point is `main.ts`